src/vmm/src/persist.rs

```diff
@@ -297,23 +297,28 @@ fn snapshot_memory_to_file(
 
 /// Validate the microVM version and translate it to its corresponding snapshot data format.
 pub fn get_snapshot_data_version(
-    version: &Option<String>,
+    maybe_fc_version_string: &Option<String>,
     version_map: &VersionMap,
     _vmm: &Vmm,
```

Comment:
Nit:
```suggestion
    vmm: &Vmm,
```

Reply:
This will lead to a build error on aarch64. Because this variable is not needed on aarch64.